define/pull.go

```diff
@@ -25,24 +25,19 @@ const (
 	// take, signalling that the source image should not be pulled from a
 	// registry.
 	PullNever
-
-	// OCI used to define the "oci" image format
-	OCI = "oci"
-	// DOCKER used to define the "docker" image format
-	DOCKER = "docker"
 )
 
 // String converts a PullPolicy into a string.
 func (p PullPolicy) String() string {
 	switch p {
 	case PullIfMissing:
-		return "PullIfMissing"
+		return "missing"
```

Comment:
Should this be "ifmissing"?

Reply:
No these names need to match the PolicyMap defined below.

```
// String converts a PullPolicy into a string.
func (p PullPolicy) String() string {
	switch p {
	case PullIfMissing:
		return "missing"
	case PullAlways:
		return "always"
	case PullIfNewer:
		return "ifnewer"
	case PullNever:
		return "never"
	}
	return fmt.Sprintf("unrecognized policy %d", p)
}
```

Reply:
Why is that?

Reply:
Specifically, why aren't we correcting the entry in the PolicyMap?

Reply:
The policy map is translating User Input into the constants.

```
man buildah pull
...
       --policy=always|missing|never

       Pull image policy. The default is missing.

              • missing:  attempt  to pull the latest image from the registries listed in registries.conf if a local image does not ex‐
                ist. Raise an error if the image is not in any listed registry and is not present locally.

              • always: Pull the image from the first registry it is found in as listed in  registries.conf.  Raise  an  error  if  not
                found in the registries, even if the image is present locally.

              • never:  do not pull the image from the registry, use only the local version. Raise an error if the image is not present
                locally.
```

Reply:
I want to pass the same names over the from podman bindings -> podman libpod handler.

Reply:
[shrugs]